microsoft-edge/webview2/concepts/visual-vs-windowed-hosting.md

```diff
@@ -0,0 +1,295 @@
+
+
+<!-- ====================================================================== -->
+## Compatibility and constraints
+
+Key compatibility limitations include the operating system and rendering in framework and non-framework applications.
+
+#### Operating system
+
+Windows 7 and Windows 8 can only do windowed hosting.
+  
+#### Rendering WebView2 in framework and non-framework applications
+
+If you're using a UI framework for your application, you should use the corresponding WebView2 element for that UI framework. If you aren't using a UI framework for your application (for example, Win32, React Native, etc.) or your UI framework doesn't have a WebView2 element, you'll have to create `CoreWebView2Controller` and render it into the desired application. 
+
+> [!NOTE]
+> If your application's UI is built using `DirectComposition` or `Windows.UI.Composition`, then you should use `CoreWebView2CompositionController`, otherwise you should use `CoreWebView2Controller`. 
+
+The `CoreWebView2Controller` properties and methods:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2Controller.CoreWebView2 Property](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2controller.corewebview2?view=webview2-dotnet-1.0.1293.44)
+* [CoreWebView2Controller.Close Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2controller.close?view=webview2-dotnet-1.0.1293.44)
+* [CoreWebView2Environment.CreateCoreWebView2ControllerAsync Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createcorewebview2controllerasync)
+
+---
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2Controller.CoreWebView2 Property](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2controller?view=webview2-1.0.1293.44#get_corewebview2)
+* [CoreWebView2Controller.Close Method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controller?view=webview2-winrt-1.0.1293.44#close)
+* [CoreWebView2Environment.CreateCoreWebView2ControllerAsync Method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment)
+
+---
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Controller::CoreWebView2 Property (get](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2controller?view=webview2-1.0.1293.44) <!--no put--> 
+* [ICoreWebView2Controller::Close Method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2controller?view=webview2-1.0.1293.44#close)
+* [CoreWebView2Environment::CreateCoreWebView2ControllerAsync Method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2environment?view=webview2-1.0.1293.44#createcorewebview2controller)
```

Comment:
```suggestion
* [CoreWebView2Environment::CreateCoreWebView2Controller Method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2environment?view=webview2-1.0.1293.44#createcorewebview2controller)
```
pls note this particular fix.  in c++ the method name lacks "async" suffix, reflect that in link text